### DISCUSSION TITLE
How to include example vignettes of how the packages can be used together

### DISCUSSION THREAD

**Discussion by epijim** [pharmaverse/pharmaverse#98] [Ideas]
After a quick chat with @statasaurus 

If we want to build vignettes showing how the packages can be used, can we host that in a way that is close to the pharmaverse. 

- [x] make a new repo to house them (for now at least, allow iteration without impacting the site)
- [ ] add content as simple `rmds` for now
- [ ] discuss and decide on how to frame them (e.g. hugo site in repo, integrate with main site, etc)
- [ ] figure out build process (e.g. gh-action to generate vignettes)
- [ ] figure out versioning (e.g. can you roll back to vignettes from earlier versions, via a tool like docat or simialar, and what is the granularity when talking about multiple packges (e.g. single environment that runs all vignettes?)

**Comment by epijim** [comment]
> make a new repo to house them (for now at least, allow iteration without impacting the site)

Done with https://github.com/pharmaverse/end_to_end_examples

**Comment by epijim** [comment]
>  add content as simple rmds for now

@lewandt - do you think we could pull some TLG vignettes from the Nest workshops and the eSub and tables working groups?

**Comment by lewandt** [comment]
@epijim I think so and I am planning to deploy the example APP to the shinyapps.io providing the link. 
I have a new example where I present the same demographic table using Tplyr, rtables and table1, plus ggplot, plotly and visR.
I use this example a lot in discussions.
